include/hpdf_version.h

```diff
@@ -0,0 +1,10 @@
+#define HPDF_MAJOR_VERSION 2
+#define HPDF_MINOR_VERSION 4
+#define HPDF_BUGFIX_VERSION 2
+#define HPDF_EXTRA_VERSION ""
+
+#define hpdf_vquote(x) #x
+#define hpdf_vexp(x) hpdf_vquote(x)
+
+#define HPDF_VERSION_TEXT (hpdf_vexp(HPDF_MAJOR_VERSION) "." hpdf_vexp(HPDF_MINOR_VERSION) "." hpdf_vexp(HPDF_BUGFIX_VERSION) HPDF_EXTRA_VERSION)
+#define HPDF_VERSION_ID ((HPDF_MAJOR_VERSION)<<16|(HPDF_MINOR_VERSION)<<8|(HPDF_BUGFIX_VERSION))
```

Comment:
I don't think this will result in the same format of HPDF_VERSION_ID used previously. Could you update it to reflect:
`
math(EXPR PROJECT_VERSION_ID "
    10000 * ${PROJECT_VERSION_MAJOR} +
    100 * ${PROJECT_VERSION_MINOR} +
    ${PROJECT_VERSION_PATCH}
")
`

Reply:
You're right, I've commited an update to fix this.